mmv1/products/artifactregistry/Repository.yaml

```diff
@@ -388,14 +461,29 @@ properties:
         properties:
           - !ruby/object:Api::Type::Enum
             name: 'publicRepository'
-            exactly_one_of:
-              - remoteRepositoryConfig.0.docker_repository.0.public_repository
             description: |-
               Address of the remote repository.
             immutable: true
+            conflicts:
```

Comment:
fyi this handles the case where a customer sets both publicRepository and customRepository, we don't have to manually handle in the pre_create hook